### PR TITLE
Refine the script to support adding startup parameters.

### PR DESCRIPTION
**Pull request by gsakun**
Refine the script to support adding startup parameters. such as 
**./ilogtail_control.sh start_and_block -enable_containerd_upper_dir_detect**

### CODE REVIEW

**Comment by yyuuttaaoo** [comment]
Looks good! But should it be -enable_containerd_upper_dir_detect=true?
However you don't have to use args. You can export enable_containerd_upper_dir_detect=true as env var to enable the feature.

**Review by yyuuttaaoo — APPROVED** [review]

**Comment by gsakun** [comment]
> Looks good! But should it be -enable_containerd_upper_dir_detect=true? However you don't have to use args. You can export enable_containerd_upper_dir_detect=true as env var to enable the feature.

e561151dc1082de444162fcde3aa925


> Looks good! But should it be -enable_containerd_upper_dir_detect=true? However you don't have to use args. You can export enable_containerd_upper_dir_detect=true as env var to enable the feature.

ok, just an example

**Comment by gsakun** [comment]
> Looks good! But should it be -enable_containerd_upper_dir_detect=true? However you don't have to use args. You can export enable_containerd_upper_dir_detect=true as env var to enable the feature.

ok,just an example

File: scripts/ilogtail_control.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ start_ilogtail() {
         local ilogtail_pid=$(load_pid)
         echo "ilogtail already started. pid: $ilogtail_pid"
     } || {
-        ($bin_file) &
+        ($bin_file $1) &
```
Comment (messixukejia) [inline scripts/ilogtail_control.sh:56]: 在logtail 启动相关文档里加下说明吧

Comment (gsakun) [reply scripts/ilogtail_control.sh]: 添加了

File: docs/cn/installation/start-with-k8s.md [hunk 2]
```diff
@@ -202,6 +202,19 @@ spec:
 wget https://raw.githubusercontent.com/alibaba/ilogtail/main/example_config/start_with_k8s/ilogtail-daemonset.yaml
 ```
 
+添加参数示例
+```
+        - name: logtail
+          command:
+          - /usr/local/ilogtail/ilogtail_control.sh
+          - start_and_block
```
Comment (yyuuttaaoo) [inline docs/cn/installation/start-with-k8s.md:210]: Did you test it?
Isn't start_and_block an arg itself?

Comment (yyuuttaaoo) [inline docs/cn/installation/start-with-k8s.md:210]: Did you test it?
Isn't start_and_block an arg itself?

**Review by yyuuttaaoo — CHANGES_REQUESTED** [review]

File: docs/cn/installation/start-with-k8s.md (hunk 2 above)
Comment (gsakun) [reply docs/cn/installation/start-with-k8s.md:210]: > Did you test it? Isn't start_and_block an arg itself?

Yes, I wrote it in the wrong place.

**Review by yyuuttaaoo — APPROVED** [review]

**Pull request merged**